apps/onlyoffice.sh

```diff
@@ -23,6 +23,51 @@ root_check
 # Nextcloud 13 is required.
 lowest_compatible_nc 13
 
+## Onlyoffice for NC 18.0.1 onwwards
+# Check for the NC-version
+CURRENTNCVERSION="$(occ_command config:system:get version | sed 's/^\(.*\..*\..*\)\..*/\1/')"
+CURRENTNCVERSION="${CURRENTNCVERSION//./}"
```

Comment:
I don't understand this?

Reply:
this gets the current version from the config file, removes everything after the 4th point (since sometimes you get 18.0.1.4) and deletes all points. So that you get eg. 1801 back.

Reply:
Have you checked the current scripts? E.g: https://github.com/nextcloud/vm/blob/master/nextcloud_update.sh#L186 :)

Reply:
yes, but I need the whole point release here. and not just the major release version...